images/rstudio-server/README.md

```diff
@@ -0,0 +1,7 @@
+# Example RStudio Server
```

Comment:
```suggestion
# RStudio Server
```

Reply:
Removed the RStudio image. Moved IDE to a sub-module